src/routes/RemediationDetails.js

```diff
@@ -209,8 +251,20 @@ class RemediationDetails extends Component {
                                             </CardBody>
                                         </Card>
                                     </StackItem>
-                                    <StackItem>
-                                        <RemediationDetailsTable remediation={ remediation } status={ this.props.selectedRemediationStatus }/>
+                                    <StackItem className='ins-c-playbookSummary__tabs'>
+                                        <Tabs activeKey={ this.state.activeTabKey } onSelect={ this.handleTabClick }>
+                                            <Tab eventKey={ 0 } title='Issues'>
+                                                <RemediationDetailsTable remediation={ remediation } status={ this.props.selectedRemediationStatus }/>
+                                            </Tab>
+                                            <Tab eventKey={ 1 } title='Activity'>
+                                                { this.state.isEntitled
+                                                    ? playbookRuns && playbookRuns.length
+                                                        ? <RemediationActivityTable remediation={ remediation } playbookRuns={ playbookRuns }/>
+                                                        : <NotConfigured/>
```

Comment:
If the playbookRuns list is empty we should show this: https://marvelapp.com/j3b9b7e/screen/63670938

`<NotConfigured/>` should only be shown when receptor is not configured. Whether receptor is configured or not can be figured out from an API call to sources API. See https://projects.engineering.redhat.com/browse/RHCLOUD-5402 for details